### ISSUE TITLE
Interface doesn't display colors

### ISSUE THREAD

**Issue by MikHulk** [cmus/cmus#1252]
Hello,

I run cmus 2.10 on freebsd. After an upgrade the interface doesn't display correctly anymore, no cursor, no graphical element just text.

I have tried to reset configuration rebuild several time the port, return to old version nothing change. I suspect an issue with ncurses. 

I will try to build cmus from the source in the next days. But if you have any idea don't hesitate. help is very welcome.

---

1) `uname -a`:
```
FreeBSD labo 14.0-CURRENT FreeBSD 14.0-CURRENT #1 main-n261958-28b498e65ab4: Sun Apr  2 21:52:11 CEST 2023     root@labo:/usr/obj/usr/src/amd64.amd64/sys/GENERIC amd64
```

2) `cmus --version`:
```
$ cmus --version
cmus v2.10.0
Copyright 2004-2006 Timo Hirvonen
Copyright 2008-2016 Various Authors
```

orginal freebsd issue:

https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270568

**Comment by MikHulk** [comment]
Building from [this commit](https://github.com/cmus/cmus/commit/ade6e2a7253d7a256e87fdac3da17c9158e6700e) worked on my system following the instruction given in the README.

I just had to set the proper audio plugin in the settings  (pulse on my case).

peace

**Comment by dr3mro** [comment]
using freebsd 14rc4 and latest cmus 
unusable due to no colors as I cannot see selection 
xfce-terminal and zsh

**Comment by gavtroy** [comment]
It's related to broken feature detection when compiling with clang. This was fixed by #1269.

**Comment by MikHulk** [comment]
I have stopped using FreeBSD. Cmus worked with a build from source but not with port version.  The issue on FreeBSD is here https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=270568
